packages/origin-backend-core/src/OrganizationInvitation.ts

```diff
@@ -2,9 +2,10 @@ import { IOrganization } from './Organization';
 import { Role } from './User';
 
 export enum OrganizationInvitationStatus {
-    Pending,
-    Rejected,
-    Accepted
+    Rejected = 1,
+    Accepted = 2,
+    Sent = 3,
```

Comment:
What's the different between Sent and Pending? What happen to existing Pending invitations?